clang/lib/CodeGen/CGExprScalar.cpp

```diff
@@ -156,6 +156,10 @@ struct BinOpInfo {
     }
     return false;
   }
+
+  /// Does the BinaryOperator have the wraps attribute?
+  /// If so, we can ellide overflow sanitizer checks.
```

Comment:
s/ellide/elide/

Reply:
fixed in [872e5d6](https://github.com/llvm/llvm-project/pull/86618/commits/872e5d6a2791c7de1268ff3d0bc090578f42067c)